src/compiler/abstract_compiler.nit

```diff
@@ -381,15 +381,15 @@ class MakefileToolchain
 
 		makefile.write """
 ifeq ($(origin CC), default)
-        CC = ccache cc
+	CC = ccache cc
```

Comment:
why changing this? tabs have a special meaning on Makefiles, one should use them wisely.